modules/imaging_browser/php/NDB_Form_imaging_browser.class.inc

```diff
@@ -486,6 +488,31 @@ class NDB_Form_Imaging_Browser extends NDB_Form
     }
 
     /**
+     * Gets candidate's scans with the same acquisitionID.
+     *
+     * @return string of FileIDs, comma separated
+     */
+    function _getFilesAcrossTimepoints($initialFileID, $initialFileAcquisitionID){
+
+        $selectResults = $this->DB->pselect("SELECT FileID FROM files ".
+            "WHERE AcquisitionProtocolID =:initialFileAcquisitionID ".
+            "AND SessionID in (SELECT s2.ID FROM session s1 RIGHT JOIN session s2 ".
+            "ON s1.CandID = s2.CandID WHERE s1.ID =:initialFileID)",
```

Comment:
@driusan yes, if I had the initial candID, but I don't, unless I wanted to regex on the file name, which I figured was not very robust.